neuralmonkey/tests/test_vocabulary.py

```diff
@@ -49,6 +49,7 @@ def test_there_and_back_self(self):
     def test_min_freq(self):
 
         vocabulary = Vocabulary()
+        vocabulary.correct_counts = True
```

Comment:
Představ si že si nainicializuješ slovník bez počtů jen ze seznamu slov a pak do něj budeš chtít přidat text.. Možná teda ani padat by to tam nemělo.